packages/-ember-data/addon/setup-container.js

```diff
@@ -41,19 +40,13 @@ function initializeStore(application) {
   }
 }
 
-function initializeDataAdapter(application) {
-  application.register('data-adapter:main', DebugAdapter);
```

Comment:
with this removed there should be an `app/data-adapter.js` file now which contains that `DebugAdapter`